### ISSUE TITLE
Android Runner

### ISSUE THREAD

**Issue by raulcraveiro** [lutris/lutris#1985]
I would love to have the possibility to play Android games via Lutris.

**Comment by strycore** [comment]
Sorry but at the moment, no satisfying solution has been found to run Android games on a Linux computer. I think the emulators need to mature a bit before we can think about supporting Android games.

**Comment by raulcraveiro** [comment]
Thank you for the answer! I hope this happens soon, because It would be amazing to play Android games like native games on Linux ❤️

**Comment by strycore** [comment]
@raulcraveiro on the other hand, I switched my Pinebook on Android and now I get plenty of Android games on a laptop, it's pretty cool!

Let me know if you ever come across good Android emulators, would be nice to have in Lutris!

**Comment by raulcraveiro** [comment]
I'll definitely let you guys know If I see something good. Thank you for the response and all the good work! ❤️
